cmd/oras/pull.go

```diff
@@ -89,6 +89,7 @@ func runPull(opts pullOptions) error {
 				fmt.Println(name)
 			}
 		}
+		fmt.Printf("pulled %s: %s\n", opts.targetRef, desc.Digest)
```

Comment:
Would recommend Capitalizing the message start. 
Also maybe break up the digest into another line. 

For reference this is how docker outputs the messages: 
```
Using default tag: latest
latest: Pulling from library/hello-world
Digest: sha256:92695bc579f31df7a63da6922075d0666e565ceccad16b59c3374d2cf4e8e50e
Status: Image is up to date for hello-world:latest
```